apps/web/playwright/booking-seats.e2e.ts

```diff
@@ -83,34 +83,55 @@ testBothBookers.describe("Booking with Seats", (bookerVariant) => {
       ],
     });
     await page.goto(`/${user.username}/${slug}`);
-    await selectFirstAvailableTimeSlotNextMonth(page);
-
-    // Kept in if statement here, since it's only temporary
-    // until the old booker isn't used anymore, and I wanted
-    // to change the test as little as possible.
-    // eslint-disable-next-line playwright/no-conditional-in-test
-    if (bookerVariant === "old-booker") {
-      await page.waitForURL((url) => {
-        return url.pathname.endsWith("/book");
-      });
-    }
 
-    const bookingUrl = page.url();
+    let bookingUrl = "";
```

Comment:
Getting url when booking has already 1 attendee and it's passed on the url for the second attendee after selecting a  slot.